src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -116,6 +116,7 @@ var HTMLDOMPropertyConfig = {
     optimum: 0,
     pattern: 0,
     placeholder: 0,
+    playsInline: HAS_BOOLEAN_VALUE,
```

Comment:
@zpao `muted` is `MUST_USE_PROPERTY` because user interaction affects its actual value (like input `value`).